leesy/index/blog.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+    <meta charset="UTF-8">
+    <title> leesy ruiz </title>
+    <link href="/Users/lisyruiz/Documents/Ada/6_week/Static-Site/leesy/style/blog.css" rel="stylesheet" type="text/css">
+    <link href="/Users/lisyruiz/Documents/Ada/6_week/Static-Site/leesy/style/normalize.css" rel="stylesheet" type="text/css">
+
+</head>
+
+<body>
+
+    <header>
+
+        <nav class="nav_container">
+
+
+            <div class="page_link">
+                <a href="index/home.html"> home </a>
+            </div>
+
+            <div class="page_link">
+                <a href="index/portfolio.html"> portfolio </a>
+            </div>
+
+            <div class="page_link">
+                <a href="index/aboutme.html"> about me </a>
+            </div>
+
+
+
```

Comment:
It seems like some of this whitespace is unnecessary. Readability is important, but I'd say there's a lot. It's true in other files as well, but I'm commenting here.